test/testdata/java_project_name.baseline

```diff
@@ -0,0 +1,166 @@
+/*
```

Comment:
This tests that a resource path of `projects/{project}` is created from a child_type of `projects/{project}/shelves/{shelf}`, even though the projects resource was never directly create anywhere